opensearch-observability/src/test/kotlin/org/opensearch/observability/PluginRestTestCase.kt

```diff
@@ -63,9 +63,12 @@ abstract class PluginRestTestCase : OpenSearchRestTestCase() {
         return true
     }
 
+    open fun preserveOpenSearchIndicesAfterTest(): Boolean = false
+
     @Throws(IOException::class)
     @After
     open fun wipeAllOpenSearchIndices() {
+        if (preserveOpenSearchIndicesAfterTest()) return
```

Comment:
When does this if condition trigger?

Reply:
this condition is set true in the IT created for bwc testing, so whenever BWC IT get's triggered this condition is triggered too